### ISSUE TITLE
R&D costs - [FEATURE]

### ISSUE THREAD

**Issue by NathanPeake** [Bit-Studios/Kapitalism#5]
Add costs to the R&D so tech nodes have a cost associated with them.

**Comment by Phonetic-Decay** [comment]
Are you taking this over? I noticed some notes on the Branch and was wondering if this might happen one day in the future.

**Comment by NathanPeake** [comment]
Yes kapitalism will be worked on again sometime after redux is released. im the original developer for kapitalism so no one is taking it over.
